### ISSUE TITLE
cmd-a (core:select-all) throws 'Uncaught ReferenceError: selection is not defined' in normal mode

### ISSUE THREAD

**Issue by gnl** [t9md/atom-vim-mode-plus#80]
When selecting the whole buffer in normal mode with core:select-all vim-mode-plus throws an uncaught error.

When in visual mode, this doesn't happen.

StackTrace:

```
ReferenceError: selection is not defined
    at VimState.module.exports.VimState.updateCursorStyle (/Users/gnl/.atom/packages/vim-mode-plus/lib/vim-state.coffee:176:32)
    at VimState.<anonymous> (/Users/gnl/.atom/packages/vim-mode-plus/lib/vim-state.coffee:197:32)
    at VimState.module.exports.VimState.showCursors (/Users/gnl/.atom/packages/vim-mode-plus/lib/vim-state.coffee:195:15)
    at /Users/gnl/.atom/packages/vim-mode-plus/lib/mode-manager.coffee:25:17
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:86:11)
    at ModeManager.activate (/Users/gnl/.atom/packages/vim-mode-plus/lib/mode-manager.coffee:77:14)
    at VimState._results.push._this.(anonymous function) [as activate] (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/node_modules/delegato/lib/delegator.js:67:61)
    at /Users/gnl/.atom/packages/vim-mode-plus/lib/vim-state.coffee:114:56
    at /Users/gnl/.atom/packages/vim-mode-plus/lib/vim-state.coffee:138:9
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:86:11)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:270:20)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:536:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:359:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:97:36)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)
```

**Comment by t9md** [comment]
Sorry for this degradation. Fix released as v0.10.1.

**Comment by gnl** [comment]
Fix confirmed, thanks!
